deployments/examples/ocis_full/collabora.yml

```diff
@@ -55,7 +55,7 @@ services:
     environment:
       aliasgroup1: https://${WOPISERVER_DOMAIN:-wopiserver.owncloud.test}:443
       DONT_GEN_SSL_CERT: "YES"
-      extra_params: --o:ssl.enable=false --o:ssl.termination=true --o:welcome.enable=false --o:net.frame_ancestors=${OCIS_DOMAIN:-ocis.owncloud.test}
+      extra_params: --o:ssl.enable=false --o:ssl.ssl_verification=${SSL_VERIFICATION:-true} --o:ssl.termination=true --o:welcome.enable=false --o:net.frame_ancestors=${OCIS_DOMAIN:-ocis.owncloud.test}
```

Comment:
> --o:ssl.enable=false

This doesn't need to be enabled to make it "secure" as Michael asked for it

Reply:
thanks 👍  I did it by analogy with `INSECURE=true` 
by default `SSL_VERIFICATION=true` `SSL_ENABLE=true` 
for development/testing is set to false